examples/BlueprintPlugin/Source/ui/src/App.js

```diff
@@ -79,8 +79,11 @@ class App extends Component {
           <Chrome
             {...styles.chrome}
             onToggleSettings={this._onToggleSettings}
-            onToggleHelp={this._onToggleHelp}>
-            <Help {...styles.contentContainer} />
+            onToggleHelp={this._onToggleHelp}
```

Comment:
I'm not sure this part needs to change?

Reply:
ok was replacing the Help tag to test the version global, would make total sense to host both the help and the version print in that same view ...but if I put  _<Help {...styles.contentContainer} />_ back I get this error (which I got also prior to my change)

```
**Blueprint Fatal Error:** uncaught: 'ScrollView must have only one child, and that child must be a ScrollView.ContentView.'
It is now unsafe to execute further code in the EcmascriptEngine.
libc++abi.dylib: terminating with uncaught exception of type std::runtime_error: uncaught: 'ScrollView must have only one child, and that child must be a ScrollView.ContentView.'


```

Reply:
Ah, yea, it depends where you're putting these things: `<Help>` is just a big `<View>` with a `<ScrollView>` inside of it. The `ScrollView` can only ever have one child, which is internally handled by setting the component of a `juce::Viewport`.

Maybe we just put the `<Text color={Colors.STROKE}>{"v" + global.__VERSION__}</Text>` inside `Help.js` as a `<SectionTitle>`?

Reply:
Anything goes for me... Just saying that that view was crashing in me also on master, does it work on yours. if it does work on your machine this means it's maybe coming from a difference in our Blueprint plugin specific package.json ,?

Reply:
Hm, that's odd, I mean the only `ScrollView` is inside `Help.js`, and that `Help` component doesn't care about any children coming in on props.

Seems a little bit like it's a problem unrelated to this PR, so I'll go ahead and merge the PR and then look into it separately. I'll comment back if I find it!